share.md

```diff
@@ -13,8 +13,8 @@ ev3dev.org Projects Page
 We currently have a [projects page] where you can browse projects that have been
 built using ev3dev. You can add your own too!
 
-{% assign project = site.categories.projects | first %}
-Here is the most recent project ([more about this project...]({{ site.github.url }}{{ project.url}})).
+{% assign project = site.categories.projects | last %}
+Here is the sample project ([more about this project...]({{ site.github.url }}{{ project.url}})).
```

Comment:
I mean in addition to. Somewhere we have instructions on how to create your own project page. Perhaps we could just include it here instead.